lib/public_key/test/public_key_SUITE.erl

```diff
@@ -646,6 +651,22 @@ encrypt_decrypt(Config) when is_list(Config) ->
     RsaEncrypted2 = public_key:encrypt_public(Msg, PublicKey),
     Msg = public_key:decrypt_private(RsaEncrypted2, PrivateKey),
     ok.
+
+%%--------------------------------------------------------------------
+encrypt_decrypt_sign_fun() ->
+    [{doc, "Test public_key:encrypt_private with user provided sign_fun"}].
+encrypt_decrypt_sign_fun(Config) when is_list(Config) ->
+    {PrivateKey, _DerKey} = erl_make_certs:gen_rsa(64),
+    #'RSAPrivateKey'{modulus=Mod, publicExponent=Exp} = PrivateKey,
+    EncryptFun = fun (PlainText, Options) ->
+        public_key:encrypt_private(PlainText, PrivateKey, Options)
+    end,
```

Comment:
pls format it as Emacs does.
![image](https://github.com/erlang/otp/assets/4620572/e78daa85-f80e-4764-a8d3-f4dac638034e)